packages/web/src/components/organisms/TriggersFormItems/ResourceTriggerSelect/ResourceTriggerSelect.tsx

```diff
@@ -24,38 +24,46 @@ import {StyledResourceOptionWrapper} from './ResourceTriggerSelect.styled';
 const {Option, OptGroup} = Select;
 
 interface ResourceTriggerSelectProps {
+  $allowTests: boolean;
+  $allowTestSuites: boolean;
```

Comment:
Better to keep them without `$`. We use `$` prefix only for styled-component's [**transient props**](https://styled-components.com/docs/api#transient-props), not for logical props.

Reply:
The only thing is that React is complaining about using `allowTests` and that we should use `allowtests`... Not sure how to skip this warning

Reply:
I see, the problem is in line 66:
```tsx
    <Select optionLabelProp="key" placeholder="Your testkube resource" showSearch {...props}>
```

The `Select` gets all the `props` down, while it should only have `rest`:

```diff
-     <Select optionLabelProp="key" placeholder="Your testkube resource" showSearch {...props}>
+     <Select optionLabelProp="key" placeholder="Your testkube resource" showSearch {...rest}>
```

Otherwise, props like `allowTests` are passed to `Select` and not recognized.